build_projects/shared-build-targets-utils/Utils/Crossgen.cs

```diff
@@ -14,6 +14,7 @@ public class Crossgen
         private string _jitVersion;
         private string _crossGenPath;
         private string _targetRID;
+        private static readonly string s_transportPackagePrefix = "transport";
```

Comment:
Why does this have to be a constant instead of just being inlined everywhere?  Do we expect to be changing the prefix?